UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs

```diff
@@ -416,6 +426,63 @@ public virtual async Task<Material> LoadMaterialAsync(int materialIndex)
 			return _assetCache.MaterialCache[materialIndex].UnityMaterialWithVertexColor;
 		}
 
+        /// <summary>
+        /// Load a Mesh from the glTF by index
+        /// </summary>
+        /// <param name="meshIndex"></param>
+        /// <returns></returns>
+        public virtual async Task<Mesh> LoadMeshAsync(int meshIndex, CancellationToken cancellationToken)
+        {
+            try
+            {
+                lock (this)
+                {
+                    if (_isRunning)
+                    {
+                        throw new GLTFLoadException("Cannot CreateMesh while GLTFSceneImporter is already running");
+                    }
+
+                    _isRunning = true;
+                }
+
+				if (_options.ThrowOnLowMemory)
```

Comment:
Tabs here.  Should be spaces.

Reply:
Actually the whole file is tabs except for this function. Updated this to be consistent.

Reply:
Oh yeah, you're right.  I had it backwards.